projects/WeTek_Core/initramfs/platform_init

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 ################################################################################
 #      This file is part of OpenELEC - http://www.openelec.tv
-#      Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
```

Comment:
As the vast majority of the content of the two `platform_init` files appears to be new and original work I'd replace the OpenELEC licence with the standard LibreELEC licence. If you don't want to put yourself as copyright then use `Team LibreELEC` - see [distro-tool](https://github.com/LibreELEC/LibreELEC.tv/blob/master/tools/distro-tool) as an example.